### DISCUSSION TITLE
lighting an Ar.js scene

### DISCUSSION THREAD

**Discussion by AdamG01** [google/model-viewer#2881] [General]
Hi guys,
I'm not a coder but am working on a personal project and was wondering if you could help me understand something..
I have created this web page with a marker for AR (I used this page to create it) [https://awesomeopensource.com/project/AR-js-org/studio](url)
This is the page :
[https://artshirtsonline.com/models/vault/index.html](url)
and this is the marker:
[https://artshirtsonline.com/wp-content/uploads/2021/10/pattern-test-Qmark_franklin-gothic.png](url)
Now it's very dark with built-in lights. Can I make a combination in code with the model viewer? 
like here: [https://modelviewer.dev/examples/lightingandenv/](url)
So the model will get its lights and reflection from an HDRI/ environment map?
Thank you so so much

**Comment by elalish** [comment]
I'm going to need a lot more detail  to help. What AR mode are you using? This should  work as  you desire by default in WebXR mode. In the  native app versions, this level of customization is not available. Also, your links are not working.
